lua/neogit/lib/git/stash.lua

```diff
@@ -125,7 +130,13 @@ function M.register(meta)
     state.stashes.items = util.map(M.list(), function(line)
       local idx, message = line:match("stash@{(%d*)}: (.*)")
 
+      ---@class StashEntry
       return {
+        rel_date = cli.log
+          .max_count(1)
+          .format("%cr")
+          .args("stash@{" .. idx .. "}")
```

Comment:
```suggestion
          .args(("stash@{%s}"):format(idx))
```

Not a major thing, but in lua strings are interned, so concatenation operations will re-hash the new string, for every `..` operator. Many successive re-hashings will have a noticeable performance hit.  While it's probable that this list will never be long enough to encounter that, I just prefer to do string formatting or `table.concat()` to join strings :)